src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php

```diff
@@ -74,6 +73,26 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $entrypointFiles[$entrypointName] = $path;
         }
 
+        if ($this->isDebug && !$input->getOption('force')) {
+            $didUnlink = false;
+            $publicDir = $this->assetsFilesystem->getDestinationPath();
+
+            foreach ($this->assetMapper->allAssets() as $asset) {
+                if (is_file($publicDir.'/'.$asset->publicPath)) {
+                    unlink($publicDir.'/'.$asset->publicPath);
```

Comment:
Maybe, the `FileSystem` component could be used here ? (With `readlink` and `remove` functions) ?

Reply:
Not without reasons. What would be yours?

Reply:
Hello, it handles issues in case of errors  like "impossible" to remove a directory and raise an IOException well formatted and `exists` handle cases like `PHP_MAXPATHLEN`, I think the lib is made to ease developer life, isn't it ? :)